nanshe_ipython.ipynb

```diff
@@ -1630,11 +1631,18 @@
         "    plot_projs.append(plot_std)\n",
         "\n",
         "\n",
+        "def gzip_compress(data, compresslevel=6):\n",
```

Comment:
This is the [same default that `zlib.compress` uses]( https://docs.python.org/3/library/zlib.html#zlib.compress ).